pkg/controller/account/ec2.go

```diff
@@ -142,25 +143,39 @@ func CreateEC2Instance(reqLogger logr.Logger, client awsclient.Client, ami strin
 		}
 		// Specify the details of the instance that you want to create.
 		runResult, runErr := client.RunInstances(&ec2.RunInstancesInput{
-			// An Amazon Linux AMI ID for t2.micro instances in the us-west-2 region
 			ImageId:      aws.String(ami),
 			InstanceType: aws.String(awsInstanceType),
 			MinCount:     aws.Int64(1),
 			MaxCount:     aws.Int64(1),
 		})
-		if runErr == nil {
-			instanceID = *runResult.Instances[0].InstanceId
-			break
-		}
 
-	}
+		// Assign instance id
+		instanceID = *runResult.Instances[0].InstanceId
```

Comment:
This needs to be set after we check for errors.

Reply:
The reason we do it here is so that in the case that we receive a timeout we have the instance ID to look at. If I put it after the tests we will never have it.